### PR TITLE
fix(aix_filesystem): remove extra param from running lsvg

### LINKED ISSUE #8151: aix_filesystem: _validate_vg not passing vg name to lsvg_cmd
### Summary

When using the community.general.aix_filesystem module to create a filesystem and passing the vg parameter to the module. The _validate_vg function fails to pass the vg variable to the in line 245

### Issue Type

Bug Report

### Component Name

aix_filesystem

### Ansible Version

```console (paste below)
$ ansible --version
ansible [core 2.15.9]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/runner/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  ansible collection location = /home/runner/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.9.18 (main, Sep 22 2023, 17:58:34) [GCC 8.5.0 20210514 (Red Hat 8.5.0-20)] (/usr/bin/python3.9)
  jinja version = 3.1.3
  libyaml = True
```

### Community.general Version

```console (paste below)
$ ansible-galaxy collection list community.general
# /usr/share/ansible/collections/ansible_collections
Collection        Version
----------------- -------
community.general 8.4.0  
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed

```

### OS / Environment

_No response_

### Steps to Reproduce

```yaml (paste below)
tasks:
    - name: Create our filesystem
      community.general.aix_filesystem:
        filesystem: "{{fs_name}}"
        size: "{{fs_size | default('1G')}}"
        vg: "{{fs_vg |default('rootvg')}}"
        state: present
```

### Expected Results

Create a new logical volume on the specified vg.

### Actual Results

```console (paste below)
FAILED! => {"changed": false, "msg": "Failed executing /usr/sbin/lsvg command."}
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by shenxianpeng**
##### SUMMARY

closes #8151

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/aix_filesystem.py

##### ADDITIONAL INFORMATION

From the [`lsvg` command examples documentation ](https://www.ibm.com/docs/en/aix/7.1?topic=l-lsvg-command#lsvg__title__6)

- To display the names of all active volume groups is command `lsvg  -o`
- To display the names of all volume groups within the system is just `lsvg`

so I removed the extra param from running `lsvg`.

### CODE REVIEW

File: plugins/modules/aix_filesystem.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def _validate_vg(module, vg):
     if rc != 0:
         module.fail_json(msg="Failed executing %s command." % lsvg_cmd)
 
-    rc, current_all_vgs, err = module.run_command([lsvg_cmd, "%s"])
+    rc, current_all_vgs, err = module.run_command([lsvg_cmd])
```
Comment (felixfontein) [inline plugins/modules/aix_filesystem.py:245]: This was introduced in 70f73f42f839ef05c81a9dd3f4c15731837400aa.

File: changelogs/fragments/8151-fix-lsvg_cmd-failed.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - aix_filesystem - fix _validate_vg not passing vg name to lsvg_cmd (https://github.com/ansible-collections/community.general/issues/8151)
```
Comment (felixfontein) [inline changelogs/fragments/8151-fix-lsvg_cmd-failed.yml:2]:
```suggestion
  - aix_filesystem - fix ``_validate_vg`` not passing VG name to ``lsvg_cmd`` (https://github.com/ansible-collections/community.general/issues/8151).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by felixfontein** [comment]
If nobody objects, I'll merge this in ~a week.

**Review by kairoaraujo — APPROVED** [review]

**Comment by felixfontein** [comment]
@shenxianpeng thanks for your contribution!
@kairoaraujo thanks for reviewing this!

**Pull request merged**